### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by chanchal16**
- Created cart management 
- add to cart
- remove from the cart
- increase quantity
- decrease quantity
- show the total price
- show total items

Preview link: https://festive-elion-38ef48.netlify.app/

### CODE REVIEW

File: src/Utils/cartUtil.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const decreaseQtyHandler = (cartDispatch,product)=>{
+    if(product.qty <= 1){
+        cartDispatch({type:'REMOVE_FROM_CART',payload:product})
+    }else{
+        cartDispatch({type:'DECREASE_QTY',payload:product})
```
Comment (OmKakatkar) [inline src/Utils/cartUtil.js:5]: Suggestion, you can make an object for the action.type constants and add 'REMOVE_FROM_CART' and 'DECREASE_QTY' as a property to that object.

Use Case: Your code editor will provide suggestion for the actions helping you code faster

File: src/pages/main.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 export {Login} from './Login';
 export {Signup} from './Signup';
 export {Products} from './Products';
-export {Wishlist} from './Wishlist'
\ No newline at end of file
+export {Wishlist} from './Wishlist';
+export {Cart} from './Cart'
```
Comment (OmKakatkar) [inline src/pages/main.js]: Its nice to see you are using barrel imports.

**Review by OmKakatkar — COMMENTED** [review]
Like the folder structure of seperating pages, components and even reducer.
It good to see that you are writing comments to functions, I need to go back do the same for my project.
Look at the suggestions, otherwise it looks good.

**Review by OmKakatkar — APPROVED** [review]

File: src/Utils/cartUtil.js (hunk 1 above)
Comment (chanchal16) [reply src/Utils/cartUtil.js:5]: Hey, Om thank you for the suggestion will refactor it later.

File: src/pages/Cart.jsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    
+  return (
+    <div className='cart-section'>
+        <div className="cart-container">
+            <div className="cart-items">
+                <div className="items-div">
+                    <div className="cart-info">
+                        <h6 className="h6">My Cart({cartState.totalItems})</h6>
+                        <span className="text-md">Deliver to:
+                            <input type="text" value="Aurangabad 431001" className="address" onChange={()=>{}} />
```
Comment (abhisheksahani-nova) [inline src/pages/Cart.jsx:19]: Remove this onChange if not needed

**Review by abhisheksahani-nova — APPROVED** [review]
Code looks clean and great work on UI , seems good to merge

**Pull request merged**